website/docs/docs/cloud/connect-data-platform/connect-snowflake.md

```diff
@@ -19,7 +19,7 @@ The following fields are required when creating a Snowflake connection
 
 ## Authentication methods
 
-This section describes the different authentication methods available for connecting dbt Cloud to Snowflake.
+This section describes the different authentication methods for connecting dbt Cloud to Snowflake. You configure Deployment environment (Production, Staging, General) credentials globally in the [**Connections**](/docs/deploy/deploy-environments#deployment-connection) area of your **Account settings**. Individual users configure their development credentials in the **Credentials** section of their user profile [**Credentials**](/docs/cloud/dbt-cloud-ide/develop-in-the-cloud#get-started-with-the-cloud-ide). 
```

Comment:
I think we can say "credentials" once and use "area" to describe both "Credentials" and "Connections"?
```suggestion
This section describes the different authentication methods for connecting dbt Cloud to Snowflake. Configure Deployment environment (Production, Staging, General) credentials globally in the [**Connections**](/docs/deploy/deploy-environments#deployment-connection) area of **Account settings**. Individual users configure their development credentials in the [**Credentials**](/docs/cloud/dbt-cloud-ide/develop-in-the-cloud#get-started-with-the-cloud-ide) area of their user profile. 
```